Description: Blank FILE_PATTERNS => no files processed
Forwarded: https://github.com/doxygen/doxygen/issues/7190
Last-Update: 2019-10-24

Index: doxygen/src/configimpl.h
===================================================================
--- doxygen.orig/src/configimpl.h
+++ doxygen/src/configimpl.h
@@ -126,6 +126,7 @@ class ConfigList : public ConfigOption
     void setWidgetType(WidgetType w) { m_widgetType = w; }
     WidgetType widgetType() const { return m_widgetType; }
     QStrList *valueRef() { return &m_value; }
+    QStrList getDefault() { return m_defaultValue; }
     void writeTemplate(FTextStream &t,bool sl,bool);
     void compareDoxyfile(FTextStream &t);
     void substEnvVars();
Index: doxygen/src/configimpl.l
===================================================================
--- doxygen.orig/src/configimpl.l
+++ doxygen/src/configimpl.l
@@ -1652,11 +1652,17 @@ void Config::checkAndCorrect()
   QStrList &filePatternList = Config_getList(FILE_PATTERNS);
   if (filePatternList.isEmpty())
   {
-      ConfigOption * opt = ConfigImpl::instance()->get("FILE_PATTERNS");
-      if (opt->kind()==ConfigOption::O_List)
+    ConfigOption * opt = ConfigImpl::instance()->get("FILE_PATTERNS");
+    if (opt->kind()==ConfigOption::O_List)
+    {
+      QStrList l = ((ConfigList*)opt)->getDefault();
+      const char *p = l.first();
+      while (p)
       {
-        ((ConfigList*)opt)->init();
+        filePatternList.append(p);
+        p = l.next();
       }
+    }
   }
 
   // add default pattern if needed
