Index: doxygen/src/htmlgen.cpp
===================================================================
--- doxygen.orig/src/htmlgen.cpp
+++ doxygen/src/htmlgen.cpp
@@ -785,6 +785,7 @@ void HtmlGenerator::init()
     mgr.copyResource("menu.js",dname);
   }
 
+  if (Config_getBool(GENERATE_TREEVIEW) || Config_getBool(SEARCHENGINE))
   {
     QFile f(dname+"/dynsections.js");
     if (f.open(IO_WriteOnly))
