Description: Import upstream patch for ghostscript 9.50 invocation
Origin: https://github.com/doxygen/doxygen/pull/7392/files
Last-Update: 2019-12-22

Index: doxygen/src/formula.cpp
===================================================================
--- doxygen.orig/src/formula.cpp
+++ doxygen/src/formula.cpp
@@ -180,8 +180,8 @@ void FormulaList::generateBitmaps(const
       // used.  
 
       char gsArgs[4096];
-      sprintf(gsArgs,"-q -g%dx%d -r%dx%dx -sDEVICE=ppmraw "
-                    "-sOutputFile=%s.pnm -dNOPAUSE -dBATCH -- %s.ps",
+      sprintf(gsArgs,"-q -g%dx%d -r%dx%d -sDEVICE=ppmraw "
+                    "-sOutputFile=%s.pnm -dNOPAUSE -dBATCH -dNOSAFER %s.ps",
                     gx,gy,(int)(scaleFactor*72),(int)(scaleFactor*72),
                     formBase.data(),formBase.data()
              );
