Description: Make the manpages reproducible when the timezone changes
Author: Paolo Greppi <paolo.greppi@libpf.com>
Last-Update: 2020-02-01

Index: doxygen/doc/CMakeLists.txt
===================================================================
--- doxygen.orig/doc/CMakeLists.txt
+++ doxygen/doc/CMakeLists.txt
@@ -12,7 +12,7 @@
 # input used in their production; they are not affected by this license.
 
 include (${TOP}/cmake/version.cmake)
-string(TIMESTAMP DATE "%d-%m-%Y")
+string(TIMESTAMP DATE "%d-%m-%Y" UTC)
 
 find_package(PythonInterp REQUIRED)
 find_program(EPSTOPDF NAMES epstopdf )
