Description: Make the changelog reproducible
Author: Paolo Greppi <paolo.greppi@libpf.com>
Last-Update: 2020-01-30

Index: doxygen/doc/changelog.doc
===================================================================
--- doxygen.orig/doc/changelog.doc
+++ doxygen/doc/changelog.doc
@@ -6534,7 +6534,7 @@ make sure you add the following:
        now be improved significantly.</li>
 <li>   Made some cosmetic changes to the HTML output (thanks to Ben Harper).</li>
 <li>   STRIP_FROM_PATH now by default strips the path from which doxygen is
-       run (i.e. $(PWD)/ on Unix)</li>
+       run (i.e. $PWD/ on Unix)</li>
 </ul>
 <h3>New features</h3>
 <ul>
@@ -6585,7 +6585,7 @@ make sure you add the following:
        Eoin MacDonell for the fix).</li>
 <li>   Grouped pages in the XML output did have the same id as their group.</li>
 <li>   Fixed problem handling environment variables inside a
-       quoted string in the config file (e.g. "$(HOME)/My Path/").</li>
+       quoted string in the config file (e.g. "$HOME/My Path/").</li>
 <li>   Using "\mainpage notitle" caused the "notitle" to appear in the treeview.</li>
 <li>   Page references where wrong in the latex output when PDF_HYPERLINKS
        was disabled.</li>
